modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,94 @@ def main():
     PELsPath = "/var/lib/phosphor-logging/extensions/pels/logs/"
     PELsArchivePath = "/var/lib/phosphor-logging/extensions/pels/logs/archive"
     inBMC = os.path.isdir(PELsPath)
-    parser = argparse.ArgumentParser(formatter_class=CustomFormatter, description="PELTools")
+    example_text = '''Examples:
+        -------------------------
```

Comment:
The only way someone would know that a -l and -a by default only include serviceable PELs is by deducing it from how line 718 is worded, so what would you think about the first 2 examples be the simple -l and -a cases?